This line in CMakeLists.txt causes trouble in recent cmake versions
--- a/build/CMakeLists.txt
+++ b/build/CMakeLists.txt
@@ -1,6 +1,5 @@
 # This is automatically generated file using PharoUnixConfig on 19 April 2013 2:37:54.162 pm
 cmake_minimum_required(VERSION 2.6.2)
-include(CMakeDetermineSystem)
 project("pharo")
 include(directories.cmake)
 message("${CMAKE_MODULE_PATH}")
